cmake/match.cmake

```diff
@@ -71,9 +71,9 @@ execute_process(
 )
 
 if(TEST_RESULT)
-    message(FATAL_ERROR "Failed (${TEST_RESULT}): The output of ${OUT_FILE} does not match ${MATCH_FILE}")
-elseif()
-    message("Passed: The output ${OUT_FILE} matches ${MATCH_FILE}")
+    message(FATAL_ERROR "Failed (${TEST_RESULT}): The output of test command '${TEST_FILE} ${TEST_ARGS}' does not match '${MATCH_FILE}'")
+else()
+    message("Passed: The output of test command '${TEST_FILE} ${TEST_ARGS}' matches '${MATCH_FILE}'")
```

Comment:
I actually prefer to have the tmp file here because I can easily navigate to it from the terminal.

Reply:
hmm... it wasn't even the full path:
```
Failed (1): The output of _matchtmpfile does not match
  /home/u105190/actions-runner/_work/unified-runtime/unified-runtime/test/conformance/sampler/sampler_adapter_level_zero.match
```

Perhaps we could have both - (full, if possible) path and the executed test command...

Reply:
Done.